.github/workflows/ci.yml

```diff
@@ -109,3 +92,31 @@ jobs:
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
           unchecked: 1,5069
+
+  clean-comments:
+    name: Clean PR Comments
+
+    runs-on: ubuntu-latest
+    needs: [ "htmlproofer", "codespell", "eip-validator", "eipw-validator" ]
+
+    steps:
+      - name: Delete Previous Comment
+        uses: izhangzhihao/delete-comment@5cd32b557f51a59d4c2cdf7124df2a0c4b7217c4
+        with: 
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          issue_number: ${{ github.event.number }}
+          delete_user_name: github-actions[bot]
+
+  comment:
+    name: Add PR Comment
+
+    runs-on: ubuntu-latest
+    if: needs.htmlproofer.result == 'failure' || needs.codespell.result == 'failure' || needs['eip-validator'].result == 'failure' || needs['eipw-validator'].result == 'failure'
+    needs: [ "htmlproofer", "codespell", "eip-validator", "eipw-validator", "clean-comments" ]
+    
+    steps:
+      - name: Comment on PR
+        uses: thollander/actions-comment-pull-request@686ab1cab89e0f715a44a0d04b9fdfdd4f33d751
+        with:
+          message: The commit ${{ github.event.pull_request.head.sha }} (as a parent of ${{ github.sha }}) contains errors. Please inspect the [Files Tab](https://github.com/ethereum/EIPs/pull/${{ github.event.number }}/files) for actionable items.
```

Comment:
In the event multiple commits get run simultaneously, and a later commit finishes before an earlier commit, will the non-deleted comment point to the latest check run?

I was thinking of adding an if-statement to my script to only comment if the check run is for the latest commit. Should we do that here too?

Reply:
> In the event multiple commits get run simultaneously, and a later commit finishes before an earlier commit, will the non-deleted comment point to the latest check run?

No, but the chance that this will happen is negligible.

Reply:
It does seem like a good idea to halt the check if it isn't against the latest commit in the PR.  At the least, it could save us some CPU time.